PurchasesTests/PurchasesTests.swift

```diff
@@ -1985,6 +1985,23 @@ class PurchasesTests: XCTestCase {
         expect(self.deviceCache.clearOfferingsCacheTimestampCount) == 0
     }
 
+    func testProxyURL() {
+        expect(RCSystemInfo.proxyURL()).to(beNil())
+        let defaultHostName = "api.revenuecat.com"
+        expect(RCSystemInfo.serverHostName()) == defaultHostName
+
+        let testURL = "test_url"
+        Purchases.proxyURL = testURL
+
+        expect(RCSystemInfo.proxyURL()) == testURL
+        expect(RCSystemInfo.serverHostName()) == testURL
+
+        Purchases.proxyURL = nil
+
+        expect(RCSystemInfo.proxyURL()).to(beNil())
+        expect(RCSystemInfo.serverHostName()) == defaultHostName
```

Comment:
this two lines are testing the same as at the beginning of the test, maybe line 1989-1991 can be removed